test/e2e_node/gcloud/gcloud.go

```diff
@@ -136,6 +136,16 @@ func (gc *gCloudClientImpl) Run(
 	go func() {
 		// Create the tmp directory
 		out, err := gc.Command("mkdir", "-p", tDir)
+
+		// Work around for gcloud flakiness - TODO: debug why gcloud sometimes cannot find credentials for some hosts
```

Comment:
This is ugly. Sigh :(
Should we instead add ssh tokens and use ssh instead of gcloud?